src/extension.ts

```diff
@@ -112,11 +112,13 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
             await node.openInPortal();
         });
         registerCommand('cosmosDB.copyConnectionString', async (actionContext: IActionContext, node?: MongoAccountTreeItem | DocDBAccountTreeItemBase) => {
+            const message = 'The connection string has been copied to the clipboard';
             if (!node) {
                 node = await ext.tree.showTreeItemPicker(accountContextValues, actionContext);
             }
 
             await copyConnectionString(node);
+            await vscode.window.showInformationMessage(message);
```

Comment:
nit: Could we remove the `await`? With the `await` it'll skew the telemetry for this action because a lot of users miss the notification, never dismiss it (which is the only time `showInformationMessage` actually returns) and telemetry won't be sent

Here's similar code: https://github.com/microsoft/vscode-cosmosdb/blob/da8b567e5418b4cc7b3c31ac05a09ab99a613fcc/src/tree/SubscriptionTreeItem.ts#L66

Reply:
@burkeholland ping 😃

Reply:
Sorry! The holidays got me, y'all. 

Removed the await.